### PR TITLE
Review App Generator Guide

### PR DESCRIPTION
**Pull request by bmuniz-daitan**
This is a WIP picked up from #6 that was reverted in #7 

Goes together with https://github.com/Danmcaires/StarlingX-App-Generator/pull/1/files

### CODE REVIEW

File: Demo.md [hunk 1]
```diff
@@ -17,88 +17,81 @@ example, local networks.
...
 
-1. We wanted to learn more about how to build a StarlingX Application.
-2. We want to make further improvements in the application to integrate
-   It with StarlingX sysinv to automatize specific nodes configurations
-   depending on the system states.
+We opted to deploy this proof of concept as a StarlingX Application. It's
+important to understand that any user workload can be deployed in many ways to
+the Kubernetes cluster(s) that StarlingX manages:
 
-But we would like to make It clear that if your application does not require a
```
Comment (tnpb-encora) [inline Demo.md:28]: do you think this piece of information would not be useful?

**Review by tnpb-encora — APPROVED** [review]
just answer my comment before approving.

File: Demo.md (hunk 1 above)
Comment (bmuniz-daitan) [reply Demo.md:28]: Not on the Demo, no. This information is already clear on the Readme of the App Gen.

File: AppGenGuide.md [hunk 2]
```diff
@@ -258,32 +258,34 @@ This means that, in order to be able to make additional configuration, one must:
...
-The static overrides contain all information that is not to be overwritten
-inside the values.yaml of the helm-chart.
+Helm chart in the `app_manifest.yaml`, a static-overrides file will be created.
+The static-overrides file contains all information that is not to be
+overwritten inside the values.yaml of the Helm chart.
 
 ### Plugins
 
-The generate will create 3 main plugins, the helm, the kustomize and the lifecycle.
+The StarlingX App Generator will create 3 main plugins: the Helm,
```
Comment (Danmcaires) [inline AppGenGuide.md]: why would helm be capitalize and the other won't?

Comment (bmuniz-daitan) [reply AppGenGuide.md]: Because Helm is the name of a tool. Maybe `kustomize` should be between backticks.

**Pull request merged**